apps/skolplattformen-app/android/app/build.gradle

```diff
@@ -157,82 +89,41 @@ android {
             keyPassword 'android'
         }
         release {
-            storeFile file('release.keystore')
-            storePassword 'skolplattform'
-            keyAlias 'skolplattform'
-            keyPassword 'skolplattform'
+            storeFile file('release.jks')
+            storePassword 'leeandseb'
+            keyAlias 'upload'
```

Comment:
Maybe choose something generic?

Reply:
Should we have what we had before? those are not the actual passwords anyway, are they?